gradle/verification-metadata.xml

```diff
@@ -521,16 +521,18 @@
             <sha256 value="e6dd072f9d3fe02a4600688380bd422bdac184caf6fe2418cfdd0934f09432aa" origin="Generated by Gradle"/>
          </artifact>
       </component>
-      <component group="com.google.guava" name="guava" version="31.0.1-jre">
-         <artifact name="guava-31.0.1-jre.jar">
-            <sha256 value="d5be94d65e87bd219fb3193ad1517baa55a3b88fc91d21cf735826ab5af087b9" origin="Generated by Gradle"/>
-         </artifact>
-      </component>
       <component group="com.google.guava" name="guava" version="31.1-jre">
          <artifact name="guava-31.1-jre.jar">
             <sha256 value="a42edc9cab792e39fe39bb94f3fca655ed157ff87a8af78e1d6ba5b07c4a00ab" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="com.google.guava" name="guava" version="32.0.1-jre">
+         <artifact name="guava-32.0.1-jre.jar">
+            <sha256 value="d5be94d65e87bd219fb3193ad1517baa55a3b88fc91d21cf735826ab5af087b9" origin="Generated by Gradle">
+               <also-trust value="bd7fa227591fb8509677d0d1122cf95158f3b8a9f45653f58281d879f6dc48c5"/>
```

Comment:
Hi @volodk85 , maybe you wanted to put the also-trust value inside the value field of the line exactly above? Because as it is, it seems to me we're trusting both the old sha256 value of the 31.0.1-jre version and the new sha256 value of the new version.

Reply:
Hi @volodk85 , thanks for the clarification! Totally makes sense for 7.17.

For the also-trust, just looking at the GitHub "Files changed", I see that the removed code from this file before was:

```
      <component group="com.google.guava" name="guava" version="31.0.1-jre">
         <artifact name="guava-31.0.1-jre.jar">
            <sha256 value="d5be94d65e87bd219fb3193ad1517baa55a3b88fc91d21cf735826ab5af087b9" origin="Generated by Gradle"/>
         </artifact>
      </component>
```

while the added code is:

```
      <component group="com.google.guava" name="guava" version="32.0.1-jre">
         <artifact name="guava-32.0.1-jre.jar">
            <sha256 value="d5be94d65e87bd219fb3193ad1517baa55a3b88fc91d21cf735826ab5af087b9" origin="Generated by Gradle">
               <also-trust value="bd7fa227591fb8509677d0d1122cf95158f3b8a9f45653f58281d879f6dc48c5"/>
            </sha256>
         </artifact>
      </component>
```

The first hash code to me seems to be from the old guava-31.0.1-jre.jar file (I didn't check character by character but they look fairly similar to me), and that is why it's odd to me. I believe the gradle tooling might have automatically put the hash code of the old version (maybe because it's still in your local cache? who knows), but I find it wrong to also trust the hash of the old version. So it'd be great if you could manually just use the hash of the new version, and remove the unnecessary also-trust. CI will tell us if there's anything wrong with what I propose.